modules/web_framework.rb

```diff
@@ -0,0 +1,107 @@
+
+require 'selenium-webdriver'
+
+class Web_framework
+  @url = nil
+  def initialize()
+    Selenium::WebDriver::Chrome::Service.driver_path=ENV['driver']
+    options = Selenium::WebDriver::Chrome::Options.new
+        options.add_argument('--headless')
+        options.add_argument('--no-sandbox')
+        options.add_argument('--disable-dev-shm-usage')
+        options.add_argument('--window-size=1920,1080')
+        @driver = Selenium::WebDriver.for :chrome ,options: options
```

Comment:
The webdriver related code can be refactored into another module/class, e.g. Driver.